pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>3.15</version>
+        <version>3.43</version>
```

Comment:
Required to avoid `Exit code: 1 - javadoc: error - The code being documented uses modules but the packages defined in http://docs.oracle.com/javase/8/docs/api/ are in the unnamed module.` with java11.

Reply:
This looks fixed in plugin-pom 3.40: https://github.com/jenkinsci/plugin-pom/commit/4081a4dcab8f0a0699e5cbdaaecd817b0a7b89a5